auth_admin_passkey/models/res_config.py

```diff
@@ -3,51 +3,56 @@
 # @author Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from openerp import api, fields, models
+from odoo import api, fields, models
+from odoo.tools import safe_eval
 
 
 class BaseConfigSettings(models.TransientModel):
     _inherit = 'base.config.settings'
 
-    # Getter / Setter Section
     @api.model
-    def get_default_auth_admin_passkey_send_to_admin(self, fields):
+    def get_default_auth_admin_passkey_send_to_admin(self, _):
```

Comment:
what is this syntax ? I mean set _ as parameter. maybe it's a convention in python world, but in odoo, it is not acceptable as _ is used for translation. I think it will be less readable.